### PR TITLE
Rename data-tid's of RadioFilterComponent

### PR DESCRIPTION
**Pull request by paudyalhari**
[Card](https://rentpath.leankit.com/card/596920021)

* Added data-tid to label
* Removed data-tid's from radio group filter

### CODE REVIEW

**Comment by paudyalhari** [comment]
Link to accidentally closed previous PR for this Fix - https://github.com/rentpath/react-ui/pull/125

File: packages/react-ui-core/src/Form/RadioGroup.js [hunk 1]
```diff
@@ -75,6 +75,7 @@ export default class RadioGroup extends Component {
 
   renderRadioButton(index, fieldProps) {
     const props = pick(this.props, ['name', 'hideInputElement', 'orientation'])
+
```
Comment (tadjohnston) [inline packages/react-ui-core/src/Form/RadioGroup.js]: 👍

File: packages/react-ui-core/src/Form/RadioGroup.js [hunk 2]
```diff
@@ -83,6 +84,7 @@ export default class RadioGroup extends Component {
         onChange={this.handleValueChange}
         {...omit(fieldProps, 'checked')}
         {...props}
+
```
Comment (tadjohnston) [inline packages/react-ui-core/src/Form/RadioGroup.js]: Please remove new line here.

File: packages/react-ui-ag/src/Filters/RadioGroupFilterCard.js [hunk 3]
```diff
@@ -62,6 +62,7 @@ export default class RadioGroupFilterCard extends Component {
       className,
       theme,
       fields,
+
```
Comment (tadjohnston) [inline packages/react-ui-ag/src/Filters/RadioGroupFilterCard.js]: Please remove the newline here.

File: packages/react-ui-core/src/Form/Label.js [hunk 4]
```diff
@@ -35,7 +35,7 @@ export default class Label extends Component {
 
     return (
       /* eslint-disable jsx-a11y/label-has-for */
-      <label {...props} className={classnames}>
+      <label data-tid="label" {...props} className={classnames}>
```
Comment (tadjohnston) [inline packages/react-ui-core/src/Form/Label.js]: Might be easier to read if the props were on separate lines (max for linter is 4 however), so this is up to you. 👍

Comment (paudyalhari) [reply packages/react-ui-core/src/Form/Label.js]: Ohk.

**Comment by nray621** [comment]
👍

**Pull request merged**